tests/System.Text.Json.Tests/JsonObjectTests.cs

```diff
@@ -20,6 +20,7 @@ public void ParseArray()
 
                 Assert.Equal(phoneNumber, "425-214-3151");
                 Assert.Equal(age, 25);
+                Assert.Equal(parsedObject.Length, 2);
```

Comment:
You fixed the order issue below, i.e. expected first, then actual. Could you please fix the order here as well? I realize you did not write all these asserts, but it would be good to be consistent (and correct)